search.py

```diff
@@ -32,6 +43,51 @@ def retrievePostingsList(file, pointer):
 
     return postingsList
 
+
+def getQueryVector(query, dictFile, postingsFile):
+    tfidf_scores = []
+    ltc_scores = []
+    query = query.split(" ")
+    queryTerms = set(query)
+    for term in queryTerms:
+        tfidf_scores.append(get_tf_idf(term, query, dictFile, postingsFile))  # get tf_idf weights according to queryTerms
+    for score in tfidf_scores:
+        ltc_scores.append(normalise(score, tfidf_scores))
+    return ltc_scores
```

Comment:
I don't think we're supposed to return all. Instead, I think we're supposed to use a priority queue and output the top K results.